modules/trustxBidAdapter.js

```diff
@@ -93,11 +94,14 @@ export const spec = {
     if (errorMessage) utils.logError(errorMessage);
     return bidResponses;
   },
-  getUserSyncs: function(syncOptions) {
+  getUserSyncs: function(syncOptions, serverResponses) {
     if (syncOptions.pixelEnabled) {
       return [{
         type: 'image',
         url: ADAPTER_SYNC_URL
+      }, {
+        type: 'image',
+        url: ADDITIONAL_SYNC_URL
```

Comment:
Can you make this configurable? Perhaps some publishers might not want the extra sync

Reply:
Technically we could. Don't publishers just restrict the amount of sync urls per adapter that they invoke?

Reply:
they do, you guys saw this doesn't fire if syncsperbidder = 1 

I thought it just threw a warning in debug mode https://github.com/prebid/Prebid.js/blob/804295aa2dae0484d67891b73fcfc401ef8244f1/src/userSync.js#L188

I don't see the line actually honoring the setting

Reply:
maybe instead of making it configurable for your adapter, just confirm that getConfig().userSync allows iframe or more than one image